src/Microsoft.DocAsCode.Common/FileAbstractLayer/PathMapping.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common
+{
+    using System;
+    using System.Collections.Immutable;
+
+    public struct PathMapping : IEquatable<PathMapping>
+    {
+        public PathMapping(RelativePath logicalPath, string physicalPath)
```

Comment:
>RelativePath logicalPath, string physicalPath [](start = 27, length = 45)

so the first one is the target relative path and the second one is the source local file path, or 
the first one is the source relative path and the second one is the target local file path, or
both usages are fine?